civihr_default_theme/templates/page/page.tpl.php

```diff
@@ -11,12 +11,18 @@
  *
  * NOTE: The #outer-wrapper element is for the mobile menu to have a way
  * both to stick at the top of the page and to be as tall as the content (necessary because of the Drupal toolbar)
+ *
+ * @var array $page
  */
 ?>
 
 <div id="outer-wrapper">
 
-  <?php require_once __DIR__ . '/../../includes/header.inc' ?>
+  <?php
+  if (!isset($page['hide_header']) || $page['hide_header'] !== TRUE) {
```

Comment:
What about `if empty($page['hide_header']) ` ?

Reply:
It would have the same effect, but I think it might be a bit clearer by saying "if it's not true" instead of "it it's empty" here.